tools/indexer/example/Cargo.toml

```diff
@@ -5,7 +5,7 @@ authors = ["Near Inc <hello@nearprotocol.com>"]
 edition = "2018"
 
 [dependencies]
-actix = "0.9"
+actix = "0.10"
```

Comment:
Yes, wasmer 1.0.1 ask for smallvec 1.6, and so does actix 0.10. With actix 0.9 it's smallvec 1.5 and cause cargo confliction. I looked up solutions and it is said if a crate is rust, it can have multiple version, but if it has c deps, it cannot, so easiest fix is bump actix

Reply:
why is this not added to `deny.toml`?

Reply:
@bowenwang1996 it seems that the nature of the issue @ailisp talks about is not related to organizational dependency duplication, rather a technical limitation of C libraries that collide in their exported symbols. I am not sure how it is related to pure-rust `smallvec` crate, though. @ailisp Did you bump into some funny compilation/runtime error or was it a failure from `cargo-deny` on CI?

Reply:
@frol the compilation error message was "unable to find a version of smallvec that satisfy blah blah", however it's not reproducible now. I had clear cargo cache since last time, maybe cargo cache scammed somehow?

Looking a little into Cargo.lock:
```
[[package]]
name = "actix"
version = "0.9.0"
source = "registry+https://github.com/rust-lang/crates.io-index"
checksum = "a4af87564ff659dee8f9981540cac9418c45e910c8072fdedd643a262a38fcaf"
dependencies = [
 ...
 "smallvec 1.6.1",
]
```

^ used to resolve to smallvec 1.5.x, but now it also becomes 1.6.1, so there is only smallvec 1.6.1 in cargo.lock, thus no compilation error 🤔

Reply:
I undo actix to 0.9 since this compilation error doesn't exist now